error/s2n_errno.h

```diff
@@ -290,7 +290,14 @@ extern __thread const char *s2n_debug_str;
 #define STRING__LINE__ STRING_(__LINE__)
 
 #define _S2N_DEBUG_LINE     "Error encountered in " __FILE__ " line " STRING__LINE__
-#define _S2N_ERROR( x )     do { s2n_debug_str = _S2N_DEBUG_LINE; s2n_errno = ( x ); s2n_calculate_stacktrace(); } while (0)
+/**
+ * Creates a new function for stacktrace with function contracts.
+ */
+#ifdef CBMC
+#    define _S2N_ERROR( x )     do { s2n_cbmc_calculate_stacktrace(x); } while (0)
```

Comment:
```suggestion
#    define _S2N_ERROR( x )     s2n_cbmc_calculate_stacktrace(x)
```

This is now a single-statement block, so we can simplify this.

Reply:
Agree with Saswat's comment.

Reply:
Implemented as suggested.